Src/BootCamp.Chapter/Program.cs

```diff
@@ -13,7 +13,7 @@ static async Task Main(string[] args)
         {
             // Put two // in front of the await to comment out Part1And2 once you solved it.
             // Make sure to note down the secret passcode before. You will need it!
-            await Part1And2();
+            //await Part1And2();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992)